mmcv/cnn/bricks/registry.py

```diff
@@ -7,7 +7,10 @@
 UPSAMPLE_LAYERS = Registry('upsample layer')
 PLUGIN_LAYERS = Registry('plugin layer')
 
+DROPOUT_LAYERS = Registry('Drop out layers')
```

Comment:
*Just Suggestions:* I may suggest that we should be careful about designing a new registry. Currently, it seems that we will have so many registries. However, their organization is confusing. For example, what's the relationship between `PLUGIN_LAYRES` and `DROPOUT_LAYERS`?

Reply:
`PLUGIN_LAYRES`  is only used in `backbone` behind the block.
https://github.com/open-mmlab/mmdetection/blob/9def292c47406fb9f27b05e61ab348809b925705/mmdet/models/backbones/resnet.py#L218